Closes #137 - Added Simple Article Header

IamManchanda opened the pull request:
Closes #137 - Added Simple Article Header
This also includes the posts and comments section
Poke @canterberrie @rafibomb for review!

Quick Codepen => http://codepen.io/IamManchanda/full/OmNwWr/

![simple-article-header](https://cloud.githubusercontent.com/assets/4970624/25268015/bd851360-2694-11e7-9b30-0c60137677d2.png)

canterberrie commented on src/building-blocks/simple-article-header/simple-article-header.html:
```diff
@@ -0,0 +1,94 @@
+  </div>
+</div>
+
+<!-- Article Comments Section -->
```
Hm.. not sure if we need this as part of this particular building block. I was imagining this building block to just be the header so that the kit is more modular and people can choose to use a different header with their body text. @rafibomb what do you think?

canterberrie commented on src/building-blocks/simple-article-header/simple-article-header.html:
```diff
@@ -0,0 +1,94 @@
+  <div class="article-post-image">
+    <div class="thumbnail">
+      <img src="https://i.imgur.com/22ue2CP.jpg">
+    </div>
```
This image isn't showing up for me in the codepen, can you double check it?

IamManchanda replied:
@canterberrie Please check now ... in the codepen .... it was that http and https issue

canterberrie replied:
Cool, it's working now =)

IamManchanda replied:
i have updated in my code too .... will push soon!

canterberrie commented:
This is a good looking building block! Left some feedback. Also, we'll need a preview image that has more padding around it so that it shows up on the site like this (not cut off on the edges!):
![screen shot 2017-04-21 at 9 26 56 am](https://cloud.githubusercontent.com/assets/20821433/25287057/c04d5762-2674-11e7-953d-04ace201afa5.png)

IamManchanda commented:
@canterberrie i have updated the padding .... also updated preview image with some editing. Is this good now!

![simple-article-header](https://cloud.githubusercontent.com/assets/4970624/25287871/bc454cc6-26e0-11e7-9a20-2c67e70f2b77.png)

IamManchanda commented:
@canterberrie @rafibomb I think the comments section should be there and that's why i also added smooth scroll but will keep the decision on you guys!

canterberrie commented:
@IamManchanda Everything looks good to me. Talked to @rafibomb about it and we think it's worth keeping the comment section in here if it looks different (right now it looks too similar to an existing BB). It should still be kept pretty simple, but here are some suggestions! 

Card based comments (credits: https://dribbble.com/shots/2069235-Feeds-Section-Card-Style)
![screen shot 2017-04-21 at 10 53 26 am](https://cloud.githubusercontent.com/assets/20821433/25289980/ceee7312-2680-11e7-8684-d1752942c972.png)
Comments with actions (credits: https://dribbble.com/shots/2470220-Iphone-Dribbble-App-Comment-Section)
![screen shot 2017-04-21 at 10 54 29 am](https://cloud.githubusercontent.com/assets/20821433/25290011/f27d8d72-2680-11e7-9bd1-25f9d3401661.png)

IamManchanda commented:
@canterberrie If this is urgent .... can we remove the comment section for now .... 
I will come back and add this back in an another PR 
What you say ??

canterberrie commented:
@IamManchanda That sounds good! I'll merge this once you remove the comment area.

IamManchanda commented:
done @canterberrie !
